# pyo3-exceptions-repro